File: src/dotnet/Ansight/Pairing/Models/DeviceDisplayProfile.cs
namespace Ansight.Pairing.Models;

public sealed class DeviceDisplayProfile
{
    public int? WidthPx { get; set; }
    public int? HeightPx { get; set; }
    public int? DensityDpi { get; set; }
    public double? RefreshRateHz { get; set; }
    public bool? HdrSupported { get; set; }
}

File: src/ios/Sources/AnsightKit/DeviceDisplayProfile.swift
import Foundation

#if canImport(Darwin)
import Darwin
#endif

#if canImport(UIKit)
import UIKit
#endif

public struct DeviceDisplayProfile: Sendable, Codable, Equatable {
    public var widthPx: Int?
    public var heightPx: Int?
    public var densityDpi: Int?
    public var refreshRateHz: Double?
    public var hdrSupported: Bool?
}
